objects/client.c

```diff
@@ -387,11 +387,11 @@
  *
  * **Signal:**
  *
- *  * *property::icon*
+ *  * *property::icon\_sizes*
```

Comment:
This is actually incorrect, but I agree that it should be this way. Anyone wants to submit a PR with the following?
```diff
diff --git a/objects/client.c b/objects/client.c
index 3adfe3ac8..297971136 100644
--- a/objects/client.c
+++ b/objects/client.c
@@ -2344,6 +2344,7 @@ client_set_icons(client_t *c, cairo_surface_array_t array)
     lua_State *L = globalconf_get_lua_State();
     luaA_object_push(L, c);
     luaA_object_emit_signal(L, -1, "property::icon", 0);
+    luaA_object_emit_signal(L, -1, "property::icon_sizes", 0);
     lua_pop(L, 1);
 }
 
```